### DISCUSSION TITLE
Photo has previously been used. Please upload a new photo.

### DISCUSSION THREAD

**Discussion by PavelUdovichenko** [community/community#38108] [Copilot Conversations]
can't get my accesses to student pack and copilot too, i'v tried to cat color resize picture but nothings helps 
![COPILOT PROBLEM](https://user-images.githubusercontent.com/25248653/199607950-0d10e489-1810-45a2-939c-b7070f62086d.jpg)

**Comment by iamcjwilliams** [comment]
@PavelUdovichenko sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
